src/BugsnagUnity/FileManager.cs

```diff
@@ -11,21 +12,30 @@ public class FileManager
 
         private static List<SessionReport> _pendingSessions = new List<SessionReport>();
 
+        private static List<Report> _pendingEvents = new List<Report>();
+
+        private static Configuration _configuration;
+
         private static string CacheDirectory
         {
             get { return Application.persistentDataPath + "/Bugsnag"; }
         }
 
-        private static Configuration _configuration;
-
-
         private static string SessionsDirectory
         {
             get { return CacheDirectory + "/Sessions"; }
         }
 
+        private static string EventsDirectory
+        {
+            get { return CacheDirectory + "/Events"; }
+        }
+
         private static string[] _cachedSessions => Directory.GetFiles(SessionsDirectory, "*.session");
 
+        private static string[] _cachedEvents => Directory.GetFiles(EventsDirectory, "*.event");
```

Comment:
C# norms + consistency with other properties in this file.

```suggestion
        private static string[] CachedSessions => Directory.GetFiles(SessionsDirectory, "*.session");

        private static string[] CachedEvents => Directory.GetFiles(EventsDirectory, "*.event");
```